### DISCUSSION TITLE
IntelliJ copilot login issue

### DISCUSSION THREAD

**Discussion by ericjbergan** [community/community#20884] [Copilot Conversations]
I was able to use copilot when it first started, but now in intellij it keeps telling me to 'login in to github and try again' when trying to use it

**Comment by amatosg** [comment]
same, it keeps telling me to login, also that I have to complete the setup but no dialog window (Copilot Settings) is available.

IntelliJ IDEA 2022.1.3 (Ultimate Edition)
Build #IU-221.5921.22, built on June 21, 2022

Copilot: 1.1.28.1744

![Captura desde 2022-07-28 12-35-37](https://user-images.githubusercontent.com/42916953/181602220-5dd3a4fe-b206-47fc-9cc2-17f45f87df3d.png)
.

**Comment by Nittarab** [comment]
Similar error here:

<img width="354" alt="image" src="https://user-images.githubusercontent.com/6181406/182790460-a28f71da-83f3-4ac5-b5a1-6fff059767f8.png">


RubyMine 2022.2
Build #RM-222.3345.111, built on July 25, 2022

Copilot: 1.1.28.1744

**Comment by jfate-sh** [comment]
Same issue for me on

Build #IU-221.6008.13, built on July 18, 2022
Runtime version: 11.0.15+10-b2043.56 x86_64

Losing out on my 60 day trial!

**Comment by Butanium** [comment]
# Versions
Intellij : 2022.1.4
Copilot plugin : 1.1.30.1887

When I try to login, nothing happens. However the vscode extension linked to the same GitHub account do work.

1
![image](https://user-images.githubusercontent.com/55806347/185437904-ef0b88fc-5f92-4c8a-899e-c032156c3f53.png)

2
![image](https://user-images.githubusercontent.com/55806347/185437922-fad8a460-499f-4862-9258-d8418e9558fc.png)

3 - after refresh
![image](https://user-images.githubusercontent.com/55806347/185437930-907ef76e-9287-441b-a9a2-f7d0cf97002f.png)

**Reply by Butanium** [reply]
Switching to Intellij 2022.2.1 doesnt solve the issue

**Reply by puradawid** [reply]
I am using IDEA 2022.1.2 (Community Edition) and have the same problem, also found out there is a log that pops up twice and then nothing else happens (so the coPilot still isn't logged within IntelliJ)

```
2022-08-21 12:09:13,369 [ 670805]   INFO - #copilot - [INFO] [auth] [2022-08-21T10:09:13.367Z] Invalid copilot token: missing token: 403 
2022-08-21 12:09:25,405 [ 682841]   INFO - #copilot - [INFO] [auth] [2022-08-21T10:09:25.404Z] Invalid copilot token: missing token: 403
```

I have tried to check other logs or search for some issues, but no luck so far. I am using Linux BTW (Ubuntu 22.04).

**Comment by ericjbergan** [comment]
Sure would be nice if one of the copilot devs would weigh in on this and maybe get us past this stuck point.

**Comment by ottonello** [comment]
I had this issue.  had an organization(I had created myself) configured on my account. When I removed it I was able to login again.
